README.md

```diff
@@ -214,6 +214,28 @@ you print in code outside of `@test`, `setup` or `teardown` functions
 must be redirected to `stderr` (`>&2`). Otherwise, the output may
 cause Bats to fail by polluting the TAP stream on `stdout`.
 
+### File descriptor 3
```

Comment:
Suggest this rewrite to keep from burying the lede:

```markdown
### File descriptor 3 (read this if Bats hangs)

Bats duplicates standard output to file descriptor 3 to separate its TAP-compliant output
(written to FD 3) from the standard output of the `run` command, which is redirected into a
temp file. A potential side effect of this mechanism is that long-running background
processes launched by the test case will cause Bats to block until they exit, since
they will inherit FD 3 without closing it. Depending on the background process, the test
may hang indefinitely until the process is killed.

**To prevent this from happening, close FD 3 explicitly when running any command that may
launch long-running child processes**, e.g. `command_name 3>- &`.
```

Reply:
I changed the title and the final bold line.